jest.config.js

```diff
@@ -1,14 +1,25 @@
 // eslint-disable-next-line
 const { createConfig } = require('@edx/frontend-build');
 
-module.exports = createConfig('jest', {
-  // setupFilesAfterEnv is used after the jest environment has been loaded.  In general this is what you want.  
+const config = createConfig('jest', {
+  // setupFilesAfterEnv is used after the jest environment has been loaded.  In general this is what you want.
   // If you want to add config BEFORE jest loads, use setupFiles instead.  
   setupFilesAfterEnv: [
     '<rootDir>/src/setupTest.js',
   ],
   coveragePathIgnorePatterns: [
     'src/setupTest.js',
     'src/i18n',
+    'src/utils/testing',
   ],
 });
+
+// The default config in @edx/frontend-build ignores all modules that are not
+// @edx, so we cannot add another pattern. We have to replace the existing
+// pattern with one that handles both @edx and @openedx.
+delete config['transformIgnorePatterns'];
+config['transformIgnorePatterns'] = [
+  '<rootDir>/node_modules/(?!@edx/|@openedx/paragon)',
```

Comment:
This is required because edx and openedx packages are not compiled down to plain (i.e., without ES6 modules) JavaScript, so tests for components containing code from these packages must be transformed before Jest can run. This is normally handled by a pattern in the Jest configuration in `@edx/frontend-build`, which contains a rule for `@edx/` packages, but not for `@openedx` packages. It seems like, at least in this version, the two were not intended to mix.

Having to do this transformation means we have to scan the packages in node_modules, which is hella slow. The first run for the added test takes ~17s on my machine. Subsequent runs are significantly faster, but still glacial compared to what a typical test should take. Short of asking Edly to re-publish their libraries, I'm not sure there is anything that can be done about that.